plugin-server/tests/main/kafka-producer-wrapper.test.ts

```diff
@@ -56,39 +56,54 @@ describe('KafkaProducerWrapper', () => {
         it('respects KAFKA_MAX_MESSAGE_BATCH_SIZE', async () => {
             await producer.queueMessage({
                 topic: 'a',
-                messages: [{ value: Buffer.alloc(450) }],
+                messages: [{ value: '1'.repeat(400) }],
             })
             await producer.queueMessage({
                 topic: 'a',
-                messages: [{ value: Buffer.alloc(40) }],
+                messages: [{ value: '1'.repeat(20) }],
             })
             expect(flushSpy).not.toHaveBeenCalled()
             expect(producer.currentBatch.length).toEqual(2)
 
             await producer.queueMessage({
                 topic: 'a',
-                messages: [{ value: Buffer.alloc(40) }],
+                messages: [{ value: '1'.repeat(40) }],
             })
 
             expect(flushSpy).toHaveBeenCalled()
 
             expect(producer.currentBatch.length).toEqual(1)
-            expect(producer.currentBatchSize).toEqual(40)
+            expect(producer.currentBatchSize).toBeGreaterThan(40)
```

Comment:
why not check for what it's now supposed to be exactly?

Reply:
Because the result is kind of random (73) since it now accounts for sizes of the keys and so on.

This gets the intent across better.

Reply:
yup, but we might mess up that function in the future to be something way too big